src/test/resources/ca/certs/01.pem

```diff
@@ -1,72 +0,0 @@
-Certificate:
```

Comment:
I think the files for 02.pem to 06.pem should also be removed.

Reply:
I think src/test/resources/client.keystore should also be removed.

Reply:
Removed the extra files, except src/test/resources/client.keystore which is needed still for the unmodified PKCS10CertificateSigningRequestTest